pkg/roachprod/roachprod.go

```diff
@@ -1604,9 +1604,9 @@ func Create(
 		for _, provider := range createVMOpts.VMProviders {
 			// TODO(DarrylWong): support zfs on other providers, see: #123775.
 			// Once done, revisit all tests that set zfs to see if they can run on non GCE.
-			if provider != gce.ProviderName {
+			if provider == azure.ProviderName {
```

Comment:
Wouldn't this previously fail for `local` clusters, but succeed with this change? Seems safer to compare against a list of supported providers.

Reply:
Good point, totally blanked on `local`; will update.